utils/profdata_merge/process.py

```diff
@@ -0,0 +1,77 @@
+# utils/profdata_merge/process.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+
+# This file contains the worker processes that watch a file queue for files and
+# merge profile data in parallel.
+
+from multiprocessing import Process
+from main import printsync
+import pipes
+import os
+import subprocess
+
+class ProfdataMergerProcess(Process):
+    def __init__(self, config, file_queue):
+        Process.__init__(self)
```

Comment:
I think `super(ProfdataMergerProcess, self).__init__()` would be better here. Using `super` mirrors [the sample code in Python's own documentation](https://docs.python.org/2/library/multiprocessing.html#using-a-remote-manager).